torch/__init__.py

```diff
@@ -367,46 +367,46 @@ def use_deterministic_algorithms(d):
         * :class:`torch.nn.ConvTranspose2d` when called on CUDA tensor
         * :class:`torch.nn.ConvTranspose3d` when called on CUDA tensor
         * :func:`torch.bmm` when called on sparse-dense CUDA tensors
-        * :func:`torch.__getitem__` backward when `self` is a CPU tensor and
-          ``indices`` is a list of tensors
-        * :func:`torch.index_put` with ``accumulate=True`` when called on a CPU
+        * :func:`torch.Tensor.__getitem__` automatic differentiation with CPU tensors
+          when the index is a list of tensors
+        * :func:`torch.Tensor.index_put` with ``accumulate=True`` when called on a CPU
           tensor
-        * :func:`torch.put` with ``accumulate=True`` when called on a CPU
+        * :func:`torch.Tensor.put_` with ``accumulate=True`` when called on a CPU
           tensor
 
     The following normally-nondeterministic operations will throw a
-    :class:`RuntimeError` when `d=True`:
-
-        * :class:`torch.nn.AvgPool3d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.AdaptiveAvgPool2d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.AdaptiveAvgPool3d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.MaxPool3d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.AdaptiveMaxPool2d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.FractionalMaxPool2d` when called on a CUDA tensor that requires grad
-        * :class:`torch.nn.FractionalMaxPool3d` when called on a CUDA tensor that requires grad
-        * :func:`torch.nn.functional.interpolate` when called on a CUDA tensor that requires grad
+    :class:`RuntimeError` when `mode=True`:
+
+        * :class:`torch.nn.AvgPool3d` automatic differentiation with CUDA tensors
```

Comment:
@mruberry, what do you think of the way I phrased this?

Reply:
I like it. I think it needs a tweak for its language to be parallel with the "with ..." language above. What about?

"... when attempting to differentiate a CUDA tensor"?